data/courses.yaml

```diff
@@ -211,19 +211,21 @@
   tags: ['intro to proof', 'mathematics']
   lean_version: 4
   summary: >
-    I have written this as a course which is completely bilingual between English and Lean,
+    This course is completely bilingual between English and Lean,
     with every example presented both ways (students also have to solve every homework problem both ways).
     It's the standard syllabus of a US "intro to proofs" course for first/second-year undergraduates,
     covering "proof techniques", good style in proof writing,
     elementary number theory, "sufficiently large", recurrence relations,
     functions, relations, and set operations.
+    The textbook "The Mechanics of Proof" (link above) was written for this course.
   experiences: >
-    The students liked using Lean and were able to learn it at the pace required.
+    The course has been taught since 2023 and is now on its fourth iteration.
+    The students like using Lean and are able to learn it at the pace required.
     I invested a lot of energy in writing custom tactics which perform exactly one step of reasoning that I would permit on paper,
     and deliberately never mentioned several more powerful tactics.
     This is very useful if the course is explicitly supposed to teach good style in proof writing.
     It lets you off the hook in grading -- you can just say "that's not enough detail -- Lean wouldn't accept it!"
-  year: 2023
+  year: 2024
```

Comment:
Should we have a means to record multiple years here? I assume duplicating the entry would not be desirable!